scripts/hms_verification/run_hms_ct_tests.sh

```diff
@@ -388,6 +389,11 @@ else
     NUM_TESTS_PASSED=0
 
     echo "Running ${TEST_SERVICE} tests..."
+    # when executed in parallel, writing to the same log file causes unintended failures when parsing the output
+    # create a new file with the service name and timestamp to avoid this issue
+    LOG_PATH=$(mktemp "/opt/cray/tests/hms_ct_test-${DATE_TIME}-${TEST_SERVICE}-XXXXXX.log")
```

Comment:
For the "/opt/cray/tests" part maybe make a LOG_PATH_BASE_DIR at the top where LOG_PATH is set. So that the log file directory is more straight forward to change.

Reply:
Implemented.